src/client/v2/algod/algod.ts

```diff
@@ -462,4 +464,36 @@ export default class AlgodClient extends ServiceClient {
   getProof(round: number, txID: string) {
     return new Proof(this.c, this.intDecoding, round, txID);
   }
+
+  /**
+   * Returns a light block header proof for a given block.
```

Comment:
I believe descriptions in the java sdk are correct, https://github.com/algorand/java-algorand-sdk/pull/360/files#diff-5b49d198d3ad575154467c557163c27eeab6cd462df1b09fb6d182b1e2f95193